presale/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+{
+    'name': "Presale Orders",
+    'version': '1.0.0',
+    'depends': ['sale_management'],
+    'data': [
+        'security/ir.model.access.csv',
+        'security/presale_security.xml',
+        'views/presale_order_views.xml',
+        'views/sale_order_views.xml',
+        'views/presale_menus.xml',
+        'data/presale_order_sequence.xml',
+        'data/email_templete.xml',
```

Comment:
Typo in "template", don't forget to rename the file too
```suggestion
        'data/email_template.xml',
```